### ISSUE TITLE
Improve doc readability

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#1019]
### Motivation for the feature

Most users will want to use a GPU so i suggest to make this tab visible by default:

![image](https://user-images.githubusercontent.com/2482071/146649579-4201b76b-8310-41a0-8d0b-43f6c36aac33.png)

Also, I find the 'all OS' vs. 'Repo installation' confusing, because these two categories are not independent (is: choosing repo installation is included in the 'all OS' case). Why not having 'Package installation (recommended)' and 'git installation'?

Other related issues:
- https://github.com/ivadomed/ivadomed/issues/996
- https://github.com/ivadomed/ivadomed/issues/728

**Comment by dyt811** [comment]
I agree the naming is a bit confusion and will be PR updating that to say `Package installation (recommended)'` instead 

So the repo installation is in theory parallel route and does not require dedicated `pip` commands (which exists in the documentation and not in the `requirement_X.txt` files, so a potential source of duplicated information and potential for getting out of sync. 

The `All OS` route is more intended for when people did not clone down the repo and have no access to ANY of the `requirements_X.txt` file. `ivadomed` PyPI Package installation only uses requirements_common.txt and default `ivadomed` package installation does not install ANY PyTorch version and the only current way to get CPU/GPU support is via as outlined Step 3.

**Comment by dyt811** [comment]
Some of the issues mentioned above have been addressed in PR1037. Let me know what you think.
